<!-- Run the fiducial_detect node -->
<launch>
  <!-- namespace for camera input -->
  <arg name="camera" default="/raspicam_node"/>
  <arg name="image" default="image"/>

  <arg name="transport" default="compressed"/>
  <arg name="fiducial_len" default="0.158"/>
  <arg name="undistort_points" default="false"/>

  <node pkg="fiducial_detect" name="fiducial_detect"
    type="fiducial_detect" output="screen" respawn="true">
    <param name="image_transport" value="$(arg transport)" />
    <param name="fiducial_len" value="$(arg fiducial_len)"/>
    <param name="publish_images" value="false"/>
    <param name="undistort_points" value="$(arg undistort_points)"/>
    <remap from="/camera/compressed" to="$(arg camera)/$(arg image)/$(arg transport)"/>
    <remap from="/camera_info" to="$(arg camera)/camera_info"/>
  </node>
</launch>
